pgscatalog_utils/scorefile/liftover.py

```diff
@@ -9,25 +9,25 @@
 
 def liftover(df: pd.DataFrame, chain_dir: str, min_lift: float, target_build: str) -> pd.DataFrame:
     """ Liftover genomic coordinates to a different genome build """
-    df = annotate_build(df, target_build)  # grab build from scoring file headers
+    df = annotate_build(df, target_build)  # get chain_target_build (e.g. in hg notation to match chain files)
 
     mapped, unmapped = pd.DataFrame(), pd.DataFrame()
-    no_liftover: pd.DataFrame = df.query('target_build == genome_build')
-    to_liftover: pd.DataFrame = df.query('target_build != genome_build')
+    no_liftover: pd.DataFrame = df.query('chain_target_build == chain_genome_build')
+    to_liftover: pd.DataFrame = df.query('chain_target_build != chain_genome_build')
 
     if no_liftover.empty:
         logger.debug("Liftover required for all scorefile variants")
     else:
         logger.debug("Skipping liftover for scorefiles with same build as target genome")
-        no_liftover[['lifted_chr', 'lifted_pos']] = no_liftover[['chr_name', 'chr_position']]  # assume col structure
+        no_liftover.loc[:,['lifted_chr', 'lifted_pos']] = no_liftover[['chr_name', 'chr_position']]  # assume col structure
```

Comment:
this change breaks liftover, the columns `lifted_chr` and `lifted_pos` contain NaN for every variant on test data.

I fixed the problem by reverting the change, is that OK?

Reply:
Yes: it was just giving that weird pandas warning about setting data in a slice when I had it the other way.

Reply:
😬 I can't  get the same warning but I changed the function to concatenate columns instead